### ISSUE TITLE
Accessibility - Country List - When Searching TalkBack Does Not Announce Countries Found

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#81]
When searching for a country on the Country screen TalkBack does not tell the user after they typed a letter that "X number of countries were found". Also, there is not an announcement of how many were found when using the "Search" button either.

**Comment by jfc3-dol** [comment]
TalkBack now announces the number of results found as you type letters and again when you tap the "search" button, so I'm going to close this issue.
